markdown/dev/howtos/git/editing-on-github/en.md

```diff
@@ -0,0 +1,63 @@
+
+All the *live* code and images are stored in the Freesewing repository.  For security, most contributors don't have permission to change the 'live' code.  Instead we must follow three steps to make changes:
+
+1. *Fork* (create a copy of) the repository.  This allows you to experiment with changes without affecting the original project.
+2. Make the edits.
+3. Save your edits and submit a *pull request* to let the admins know about your changes.
+
+The Github documentation provides a more [detailed explanation](https://docs.github.com/en/get-started/quickstart/fork-a-repo) of this.
+
+## Fork the repository
+- On Github.com, navigate to the freesewing [repository](https://github.com/freesewing/freesewing)
+- In the top-right corner of the page, click **Fork**
+![Fork button](fork_button.png)
+- It is helpful to give your fork a name to help you and others know what has changed. E.g. if you are going to edit the Simone pattern instructions to include info on using velcro fastenings, you could call your fork "Simone-Velcro".
+- Please also include a brief Description to help the maintainers know what you are changing. In this example, you might want to write: "Updated Simone instructions with option for velcro fastenings."
+- Click the "Create fork" button.
+![Create fork button](fork-create-button.png)
+- Create a *Branch* where you will make your edits.  Click the drop-down "Develop" menu button at the top-left of the page and give your branch a meaningful, brief name, e.g.
+![Create Branch button](create-branch.png)
+<Note>
+Double-check that the "Develop" drop-down menu displays the name of the branch you just created.
+</Note>
+
+## Make the edits
+- Navigate to the folder containing the file that you want to edit.
+<Note>
+To help you understand how the code is organised, here is the path to the Simone Instructions:
+```freesewing/markdown/org/docs/patterns/simone/instructions/
+```
+This folder contains several files, one for each language.
+![Path to Simone instructions](simone-instructions-path.png)
+</Note>
+- Select on the file you want to update. In this example, we will edit the English language instructions. 
+- Click on the *Edit* button (pencil icon).
+<Note>
+Clicking *Preview* will show what your changes will look like.
+![Edit and preview buttons](edit-preview.png)
+</Note>
+
+## Save your edits and submit a pull request
+When you are happy with your edits:
+- Scroll to the bottom of the page and enter a brief title and description of the changes you made.
+- Click the *Commit* button.
+![Commit changes](commit-changes.png)
+
+Now you will create a *pull request* to submit your edits and let the admins know your changes are ready for review:
+- Navigate to the repository where you created your fork.
+- Above the list of files, click the *Pull request* button.
```

Comment:
This is where those remarks about giving it a description (above under fork) should go